app-admin/supervisor/supervisor-4.2.5_p20250108.ebuild

```diff
@@ -1,24 +1,34 @@
-# Copyright 1999-2024 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
 DISTUTILS_USE_PEP517=setuptools
-PYTHON_COMPAT=( pypy3 python3_{9..11} )
+PYTHON_COMPAT=( pypy3 python3_{10..13} )
 # xml.etree.ElementTree module required.
 PYTHON_REQ_USE="xml(+)"
 
 inherit distutils-r1 systemd pypi
```

Comment:
We don't need pypi any more, do we?

I'll be changing your files a little before applying the commit.

Reply:
For the next pypi release. SRC_URI and S can be overridden with pypi.

Reply:
Sure but for the current release we're not using pypi at all so why leave it?